index.html

```diff
@@ -1925,37 +1925,53 @@ <h4>Error Responses</h4>
           </span>
         </p>
       </section>
-    </section>
+      </section>
+
+
+
 
     <!-- External Representation -->
     <section id="external-representation">
       <h2 id="external-td-representations">External TD
         representations</h2>
       <p>
-        <span class="rfc2119-assertion" id="profile-5.3-exteanal-td-representations-1">
+        <span class="rfc2119-assertion" id="profile-5.3-external-td-representations-1">
         The default representation is JSON. Semantic
         annotations based on JSON-LD MAY be present but are not
         required to perform all interactions with the thing
         instance.
         </span>
       </p>
 
-        <section id="canonical-representation">
-          <h2 id="canonical-td-representations">Canonical TD
-            representation</h2>
-          <p>
-            A canonical representation serves multiple purposes.
-            It is simplifying the parsing process, enables to identify
-            equivalent TDs by simple string comparisons.
-            Furthermore it allows the use of a simple signing mechanism,
-            such as <a href="https://w3c-ccg.github.io/ld-proofs/">Linked Data Proofs</a> or JSON Web Signatures [[RFC7515]] and enables
-            identity checks on encrypted TDs.
-          </p>
-	  <p>
-	     The canonical JSON representation format of a TD adopts the JSON Canonicalization Scheme (JCS) defined by
-	     <a href="https://tools.ietf.org/html/rfc8785>RFC-8785</a> including errata <a href="https://www.rfc-editor.org/errata/eid6292">6292</a>.
-	  </p>
-      </section>
+      <h3 id="character encoding">Character Encoding</h3>
+      <p>
+        <span class="rfc2119-assertion" id="profile-5.3.1-external-td-representations-2">
+        To provide interoperability and in alignment with the latest version 
+        of the JSON standard [[rfc8259]],
+        the JSON representation of a thing description MUST be encoded using UTF-8 [[RFC3629]].   
+        </span>       
+      </p>
+
+    </p>
+    <h3 id="canonical-td-representations">Canonical TD
+      representation</h3>
+    <p>
+      A canonical representation serves multiple purposes.
+      It is simplifying the parsing process, enables to identify
+      equivalent TDs by simple string comparisons.
+      Furthermore it allows the use of a simple signing mechanism,
+      such as <a href="https://w3c-ccg.github.io/ld-proofs/">Linked Data Proofs</a> or JSON Web Signatures [[RFC7515]] and
+      enables
+      identity checks on encrypted TDs.
+    </p>
+    <p>
+      The canonical JSON representation format of a TD adopts the JSON Canonicalization Scheme (JCS) defined by
+      <a href="https://tools.ietf.org/html/rfc8785>RFC-8785</a> including errata <a href="
+        https://www.rfc-editor.org/errata/eid6292">6292</a>.
+    </p>
```

Comment:
What is the purpose of describing canonical representation here, is it not sufficient that this is defined in the [Thing Description](https://w3c.github.io/wot-thing-description/#canonicalization-serialization-json) specification? If necessary that could be referenced from this specification.

As an aside, something that I do think could be useful in the WoT Profile specification would be to provide an example Thing Description in the Protocol Binding section of the Core Profile, then later provide an informative canonical version of that Thing Description with all the defaults from the profile applied (e.g. with all operations, HTTP methods, headers, response codes etc. filled out).

Reply:
This is not a new section, it just gets hilighted in your diff.
If you look at the HTML diff it is not highlighted.
https://pr-preview.s3.amazonaws.com/w3c/wot-profile/86/f1ec8e2...ac4a244.html#external-td-representations

This text is older than the TD section on canonical TDs, when it was written there was nothing in the TD.
I did not want to remove it yet, since (as you state) we should reference the corresponding TD paragraph.